rdt/transformers/__init__.py

```diff
@@ -100,6 +100,8 @@ def get_transformer_name(transformer):
     'pii': AnonymizedFaker(),
 }
 
+MULTI_COLUMN_TRANSFORMERS = {}
```

Comment:
How will this be different than the current dictionary?

Reply:
Right now this dictionary is meant to map an argument (the anonymization level) to the address transformer to use. I think we should just call it `ADDRESS_TRANSFORMERS`. That's more clear and there's no guarantee other multi column transformers will have defaults set by a parameter anyway. If they do we can update